scripts/globals/items/bottle_of_ascetics_gambir.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------------
+-- ID: 5842
+-- Item: Bottle of Ascetic's Gambir
+-- Item Effect: Temporarily increases magic atk power and acc
+-----------------------------------------
+require("scripts/globals/status")
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0
+    if target:hasStatusEffect(dsp.effect.MEDICINE) then
```

Comment:
this does not happen on retail

Reply:
oh ok, i just thought i better check for it, i can get rid of them no problem

Reply:
when its a temp item, start with the assumption the usual rules ain't gonna apply

Reply:
done